src/components/CivicLikerV3/Dashboard.vue

```diff
@@ -1,39 +1,40 @@
 <template>
-  <div v-if="!validatorAddress" class="p-32">
-    <Spinner class="mx-auto" />
-  </div>
-  <CivicLikerV3PureDashboard
-    v-else
-    :status="status"
-    :is-signed-in="!!getUserId"
-    :avatar-src="getUserInfo && getUserInfo.avatar"
-    :staking-validator-name="validatorName"
-    :staking-management-url="stakingManagementURL"
-    :staking-amount="stakingAmount"
-    :staking-amount-target="stakingAmountTarget"
-    :active-since="activeSince"
-    @sign-in="handleSignIn"
-  />
+  <Transition name="fade" mode="out-in">
+    <div
+      v-if="!validatorAddress"
+      key="loading"
+      class="flex items-center justify-center min-h-[180px]"
+    >
+      <Spinner class="mx-auto" />
+    </div>
+    <CivicLikerV3PureDashboard
+      v-else
+      key="dashboard"
+      :status="status"
+      :is-signed-in="!!getAddress"
```

Comment:
Do we have UI that tells wallet user with enough stake to register a Liker ID to activate civic liker? This would be a case previously did not exist